CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## next
+- Adding NineTileBox widget
 - Fixing compilations errors regarding changes on `box2_flame`
```

Comment:
this was deployed, right? this diff is weird, please take a look

Reply:
yep, I may have messed up when re basing my branch.